sigridci/sigridci-feedback-template.html

```diff
@@ -308,6 +323,24 @@ <h3>Passed?</h3>
                     </div>
                 </div>
             </div>
+            
+            <div class="explanation tile">
+                Sigrid helps you to write maintainable code. It provides you with a definition of done for code 
+                quality, by comparing your system against the 6,000+ systems in our benchmark.
+                <ul>
+                    <li><strong>Trusted:</strong> Sigrid checks your code using SIG's
+                    <a href="https://www.softwareimprovementgroup.com/methodologies/iso-iec-25010-2011-standard/">
+                    maintainability quality model</a>, which is based on the ISO 25010 standard for software product 
+                    quality.</li>
+                    <li><strong>Holistic</strong>: SIG's maintainability model covers multiple metrics and
+                    technologies, then uses the results to calculate a single overall rating that indicates your 
+                    system's maintainability.</li>
+                    <li><strong>Pragmatic:</strong> Sigrid does not reject your code if a solitary issue was found. 
+                    As long as you meet the maintainability target overall, it's OK to have some minor issues.</li>
```

Comment:
I would change "maintainability target overall" to "overall maintainability target", or if you want to emphasise specifically that you reach the "maintainability target" over all the files that are checked I would change to something like "maintainability target over the whole codebase" or instead of "codebase" something like "all code considered" if you want to have it more open for CI checking only the affected files.